code/game/objects/items/stacks/stack_recipe.dm

```diff
@@ -35,6 +35,35 @@
 		result.color = S.color
 	..()
 
+
+/datum/stack_recipe/p_tile
```

Comment:
Would it be better to implement these all as `/datum/stack_recipe/dangerous`, to prevent repetition? Also, wouldn't it be good to add a special fingerprint to the plasma statues that can be displayed upon ignition?